api/handlers.go

```diff
@@ -182,6 +233,38 @@ func (api *API) userStatsGET(w http.ResponseWriter, req *http.Request, _ httprou
 	api.WriteJSON(w, us)
 }
 
+// userPOST creates a new user.
+func (api *API) userPOST(w http.ResponseWriter, req *http.Request, _ httprouter.Params) {
```

Comment:
REQ: Where are the tests for the methods in this file? I don't see updates for the new methods.

Reply:
We don't have any tests for the handlers. I'm not sure what would be a good approach to testing them. I don't want to expose them and I don't see how I can test them without the full integration setup. Maybe I can create exported test methods for them within this package and then have those exit right away if the testing type is not integration. And then I can call them from a test in the integration suite.

While this sounds like a good idea, I'm not sure it's what we should be doing right now. Do you mind if I  punt this into a F/U?

Reply:
Follow up is fine. We should definitely figure out how to extend the testing so that it hits mongo. I'm sure there are plenty of solutions.

Reply:
Yes, definitely. We already have a set of integration tests that run against a clean Mongo in a docker container. We just need another set which acts as an end-to-end. Should be simple enough to set up, it will just take some time.

F/U: #69